summarize_from_feedback_details/ppo.py

```diff
@@ -47,7 +47,7 @@ class PpoHParams:
     cliprange_value: float = 0.2
     gamma: float = 1
     lam: float = 0.95
-    whiten_rewards: bool = True
+    whiten_rewards: bool = False
```

Comment:
Since in the paper we find whiten rewards to be harmful, here we set it to False by default.